src/tests/unit_tests/test_config.py

```diff
@@ -0,0 +1,32 @@
+import pytest
```

Comment:
We should name the test file so that it corresponds to the file being tested.
So in this case I suggest naming it `src/tests/unit_tests/sherpa_ai/test_config.py`.

This will make it easier to set up an automated test runner later, which runs just the tests that are necessary after each file change.

Reply:
This test folder is specific to test files in`sherpa_ai`. Maybe a good idea would be grouping the files by module: `src/tests/unit_tests/config/test_task_config.py`